File: Codeforces/1. ICPC Assiut University Community/Sheet #1 (Data type - Conditions)/U. Float or int/main.cs
// https://codeforces.com/group/MWSDmqGsZm/contest/219158/problem/R

using System;

namespace Codeforces.Assuit.Sheet1
{
    class ProblemR
    {
        static void Main(string[] args)
        {
            float n = float.Parse(Console.ReadLine());
            short nn = (short)n;
            if (nn == n)
            {
                Console.WriteLine("int " + nn);
            }
            else
            {
                Console.WriteLine($"float {nn} {n - nn}");
            }
        }
    }
}

File: Codeforces/1. ICPC Assiut University Community/Sheet #1 (Data type - Conditions)/U. Float or int/main.cpp
// https://codeforces.com/group/MWSDmqGsZm/contest/219158/problem/U
#include<iostream>
using namespace std;
 
int main(){
    float n;
    cin >> n;
    short nn = n;

    if(nn == n) {
        cout << "int " << nn << endl;
    } else {
        cout << "float " << nn << " " << n - nn << endl;
    }
}
